PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkBatchJobConfigurableModel.java

```diff
@@ -39,6 +39,8 @@
     @Transient
```

Comment:
>@Transient [](start = 4, length = 10)

Why this is transient? I thought we are serializing and deserializing SparkSubmitModel

Reply:
SparkSubmitModel is a special field, which should be handled individually.